docs/development.md

```diff
@@ -379,3 +380,15 @@ vagrant up centos7-dynflow-devel
 ```
 
 In the vagrant box, the dynflow repository is cloned to `/home/vagrant/dynflow`.
+
+## Smoker
+
+The testing tool [smoker](https://github.com/theforeman/smoker) can be set up with the `centos7-foreman-smoker` box and tests can be run against a separate Foreman/Katello instance.
+
+To use:
+1. Follow the example box definition in `vagrant/boxes.d/99-local.yaml.example` and update the `smoker_base_url` variable. 
+2. Run `vagrant up centos7-foreman-smoker`. With `pytest_local_env` set to true, the box will be set up with pytest and the smoker repository will be cloned to the `vagrant` user's home directory, but the tests will not be run by the playbook.
```

Comment:
The notes about pytest_local_env should be part of step 1 because you'll want to be aware of and potentially modify it before running `vagrant up`